frontend/packages/kubevirt-plugin/src/components/modals/create-vm-wizard.tsx

```diff
@@ -78,7 +78,8 @@ export const openCreateVmWizard = (activeNamespace, createTemplate = false) => {
     return {
       selectedNamespace,
       templates,
-
+      templatesLoaded:
```

Comment:
I don't think we need this specific field. Firehose set's `loaded` to `true` if all non-optional resources are finished loading. I think we can postpone rendering based on this common `loaded` props until we have all data ready.

Reply:
`loaded` prop is not passed through `createModalResourceLauncher`. This is more granular approach and we do not have to wait for loading of e.g. namespace.

Reply:
ok. The wizard is recently under refactoring and `loaded` or `error` states will be newly handled in the wizard. So good enough for the time being.